xfce4-desktop/xfce4-desktop.yaml

```diff
@@ -0,0 +1,17 @@
+version: v1
+jobs:
+  xfce4-desktop:
+    env:
+      - HOME=/tmp/home
+    image:
+      uri: oras://repository.ciq.com/internal-hpc-apps-testing/sifs/xfce4-desktop.sif:latest
```

Comment:
I would recommend adding `username: ciq` and `password: <mountain-access-key>` here.

Reply:
I don't think you can do `username:` and `password:` here any more, now that it's all unified under `secret:`; and, with that, I don't think there's a meaningful default or placeholder value that can be supplied. But correct me if I've missed something.